packages/devtools_app/lib/src/app.dart

```diff
@@ -112,21 +112,21 @@ class DevToolsAppState extends State<DevToolsApp> with AutoDisposeMixin {
     });
 
     _isDarkThemeEnabled = preferences.darkModeTheme.value;
-    preferences.darkModeTheme.addListener(() {
+    addAutoDisposeListener(preferences.darkModeTheme, () {
```

Comment:
Fyi @polina-c this caught some cases where were adding listeners but never removing.